hopp/tools/design/wind/power_curve_tools.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import copy
+
+def plot_power_curve(wind_speeds_ms, cp_curve, ct_curve):
+    """Plot Cp and Ct curve per wind speed.
+
+    Args:
+        wind_speeds_ms (List | np.ndarray): list of wind speeds in m/s
+        cp_curve (List | np.ndarray): power curve coefficients (Cp) at each wind speed in `wind_speeds_ms`
+        ct_curve (_type_): thrust curve coefficients (Ct) at each wind speed in `wind_speeds_ms`
+    """
+    fig1 = plt.figure()
+    plt.plot(wind_speeds_ms, ct_curve, label="Coeff of Thrust")
+    plt.plot(wind_speeds_ms, cp_curve, label = "Coeff of Power")
+    plt.legend()
+    plt.xlabel("Wind Speed [m/s]")
+    plt.show()
+
+
+def pad_power_curve(wind_speed, curve, v_min = 0.0, v_max = 50.0):
```

Comment:
According to the rest of the function, `wind_speed` should be `wind_speed_ms`. I think `wind_speed` would be preferable because m/s is the industry standard, and it's defined in the docstring.

Reply:
`v_min` and `v_max` are also naming conventions for parts of matplotlib, and aren't being used in that context, could we switch to a more indicative name such as `ws_min`/`ws_max` or `wind_speeed_start`/`wind_speeed_end`?